proposals/lip-0071.md

```diff
@@ -303,21 +304,22 @@ This function is used to estimate the expected rewards for a validator assuming
 ##### Execution
 
 ```python
-def getExpectedValidatorRewards(validatorAddress: Address, height: uint32) -> tuple[uint64, uint64, uint64, uint64]:
+def getExpectedValidatorRewards(validatorAddress: Address) -> tuple[uint64, uint64, uint64, uint64]:
     validatorParams = Validators.getValidatorParams()
+    height = current height of the last block
```

Comment:
Why do you remove the height here? It seems to be used in line 317 below.

Reply:
I add it here. In the previous version of PR the height was a parameter of the function, I have moved it into the function.

Reply:
I keep the endpoint format consistent with how I defined endpoints for DEX. Many of them also required current height as an argument, see e.g. https://github.com/LiskHQ/lisk-dex-specs/blob/main/specifications/0001.md?plain=1#L1168.
I think this function signature is better since we never explicitly mention any of the data that is passed in the context. We just always assume that we can access it in the function itself. This is why I leave the function as is.

Reply:
Ok, I will not insist, fine for me if @janhack agrees as well..